### PR TITLE
Fix layouts and host for GMOSSP amp-ads

### PR DESCRIPTION
**Pull request by t-kino**
Could you merge 'Fix layouts and host for GMOSSP amp-ads' into ampproject/amphtml master branch?

We have a Google Corporate CLA.
Our CLA: GMO AD Marketing Inc.
Our company name: GMO AD Marketing Inc.

Thank you in advance

### CODE REVIEW

**Comment by t-kino** [comment]
I signed it!

**Comment by t-kino** [comment]
Could @keithwrightbos review my pull request?

**Comment by t-kino** [comment]
Hi @jridgewell, @keithwrightbos,

Could you review my pull request?
Just two business days have passed already.

Please make sure everything is done smoothly.
Could you tell me if I made any mistakes about procedures?

Best regards,

File: extensions/amp-ad-network-gmossp-impl/0.1/amp-ad-network-gmossp-impl.js [hunk 1]
```diff
@@ -23,7 +23,9 @@ import {startsWith} from '../../../src/string';
  * @type {string}
  * @private
  */
-const GMOSSP_BASE_URL_ = 'https://sp.gmossp-sp.jp';
+const GMOSSP_BASE_URL_ = 'https://sp.gmossp-sp.jp/';
+
+const GMOSSP_BASE_A4A_URL_ = 'https://amp.sp.gmossp-sp.jp/_a4a/';
```
Comment (keithwrightbos) [inline extensions/amp-ad-network-gmossp-impl/0.1/amp-ad-network-gmossp-impl.js]: typedoc?

File: extensions/amp-ad-network-gmossp-impl/0.1/amp-ad-network-gmossp-impl.js [hunk 2]
```diff
@@ -49,7 +51,8 @@ export class AmpAdNetworkGmosspImpl extends AmpA4A {
 
   /** @override */
   getAdUrl() {
-    return this.element.getAttribute('src');
+    return this.element.getAttribute('src').replace(GMOSSP_BASE_URL_,
+        GMOSSP_BASE_A4A_URL_);
```
Comment (keithwrightbos) [inline extensions/amp-ad-network-gmossp-impl/0.1/amp-ad-network-gmossp-impl.js]: I noticed that isValidElement expects the old url format.  Do you want to update to also allow a4a path?

Comment (t-kino) [inline extensions/amp-ad-network-gmossp-impl/0.1/amp-ad-network-gmossp-impl.js]: Yes, I do.

**Comment by t-kino** [comment]
@keithwrightbos
Thank you for your review. Could you check this again?

File: extensions/amp-ad-network-gmossp-impl/0.1/amp-ad-network-gmossp-impl.js (hunk 2 above)
Comment (keithwrightbos) [inline extensions/amp-ad-network-gmossp-impl/0.1/amp-ad-network-gmossp-impl.js]: Then you need to modify isValidElement to allow either GMOSSP_BASE_A4A_URL_ or GMOSSP_BASE_URL_.  You will also need to modify extensions/amp-ad-network-gmossp-impl/0.1/gmossp-a4a-config#gmosspIsA4AEnabled

**Comment by t-kino** [comment]
@keithwrightbos
Thanks for your advice. I modified as above.
Is this alright with you?

**Comment by matsumoto-kouichi** [comment]
Hi @jridgewell, @keithwrightbos,
I'm @matsumoto-kouichi and I'm a boss of @t-kino.
I would be grateful if you could please check this.

**Pull request merged**